tests/unit/dummy_data/test_query_info.py

```diff
@@ -54,7 +54,7 @@ def test_query_info_from_variable_definitions():
                     "code": ColumnInfo(
                         name="code",
                         type=str,
-                        values_used={"abc00"},
+                        _values_used={"abc00"},
```

Comment:
I questioned whether there was a better approach in [this Slack thread][1]. Outside tests, we'd call `ColumnInfo.record_value` rather than `ColumnInfo._values_used.add`. However, passing the value of an internal attribute like this seems a bit unpleasant.

[1]: https://bennettoxford.slack.com/archives/C03FB777L1M/p1686143435147369

Reply:
Ah I see. I don't think it's unreasonable for test code to use internal things as part of its setup. But an alternative would be to leave the attribute with its old name, `values_used` and call the property something like `all_values_used`.

Reply:
I think I'm going to leave this as-is, because it's test code. Thank you for highlighting an alternative.